lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/task/RunningTaskImpl.java

```diff
@@ -107,7 +107,7 @@ public void close(final byte[] message) {
           .build();
       this.evaluatorManager.sendContextControlMessage(contextControlProto);
     } else {
-      throw new RuntimeException("Trying to send a message to a Task that is no longer RUNNING.");
+      LOG.log(Level.FINE, "Ignoring call to .close(byte[] message) because the task is no longer RUNNING.");
```

Comment:
1) Please consider increasing the level of logging to INFO. AFAIK, that's REEF default level of logging, and we probably  want all synchronization issues to appear in the logs by default.
2) Could you please refer to the original JIRA issue in the log message, e.g., "See REEF-1503 for discussion"  ? The current message lacks the info on why it is safe to ignore the call to close(), which might confuse a reader.